benchmarks/logistic_regression/README.md

```diff
@@ -0,0 +1,99 @@
+# Logistic regression
+
+## Final results
+
+| Model         | Track    |   Accuracy |       F1 | Time                   | Memory   |
+|:--------------|:---------|-----------:|---------:|:-----------------------|:---------|
+| PyTorch       | Bananas  |   0.54566  | 0.202297 | 0 days 00:00:02.354718 | 16.59 KB |
+| PyTorch       | Phishing |   0.814667 | 0.762068 | 0 days 00:00:00.587769 | 16.59 KB |
+| River         | Bananas  |   0.543019 | 0.195349 | 0 days 00:00:00.508606 | 4.12 KB  |
+| River         | Phishing |   0.888    | 0.872263 | 0 days 00:00:00.155899 | 5.48 KB  |
+| Vowpal Wabbit | Bananas  |   0.547075 | 0.160224 | 0 days 00:00:00.556017 | 388 B    |
+| Vowpal Wabbit | Phishing |   0.8044   | 0.741133 | 0 days 00:00:00.160629 | 388 B    |
```

Comment:
Just a random thought: does the memory measurements here are correct? I think this might be related to Cython. What do you think? As more and more portions of our codebase are implemented in Cython, we might think of a generic and effective way to measure such structures' memory usage.

Reply:
The measurements are correct for River. They're incorrect for Vowpal Wabbit but that's down to the way their Python package works.